### DISCUSSION TITLE
GDRR solution

### DISCUSSION THREAD

**Discussion by alextseitlin** [vercel/next.js#46476] [Ideas]
### Describe the feature you'd like to request

This is very important for us in Europe. Would be nice to have a solution to GDRR on nextjs 13

### Describe the solution you'd like

Easy and fast solution

### Describe alternatives you've considered

None

**Comment by timneutkens** [comment]
It's unclear what you mean by this request in particular? Are you asking for a built-in cookie banner or something completely different?

**Comment by balazsorban44** [comment]
Could you elaborate on what you mean by GDPR solution?

There's an anonymous telemetry for when running builds, but that's fully optional https://nextjs.org/telemetry#how-do-i-opt-out

Other than that, Next.js is just a framework and you host/deploy it as you want. Check out the self-hosting documentation https://nextjs.org/docs/deployment#self-hosting

Hope this helps!
